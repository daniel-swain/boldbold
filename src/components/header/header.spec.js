/* eslint-disable */
import React from 'react';
import Header from './index';

describe('<Header />', () => {
  let wrapper;

  beforeAll(() => {
    wrapper = shallow(<Header />);
  });

  it('should render properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
