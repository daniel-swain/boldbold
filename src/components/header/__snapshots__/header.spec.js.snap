// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render properly 1`] = `
<div
  className="header"
>
  <span
    className="title"
  >
    cmon step it up
  </span>
</div>
`;
